### DISCUSSION TITLE
Display all attachments in one table column

### DISCUSSION THREAD

**Discussion by joeytzy** [Budibase/budibase#12380] [Help] [answered]
i have a table in budibase db to store files and attachments. My question is, how do i use repeater to display all attachments from a single column from a single row data as shown below,

![image](https://github.com/Budibase/budibase/assets/44018390/0d89953f-4e65-4824-8e43-3430643e2149)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @joeytzy,

I'm going to link this discussion i commented on earlier. https://github.com/Budibase/budibase/discussions/12394

The key part that will help your case is the example shown within the community discord channel [here](https://discord.com/channels/733030666647765003/1153069081029787759/1153069081029787759).

I hope this helps, in this example i've demonstrated how you can display all images stored within the design area.
